airflow/utils/dag_processing.py

```diff
@@ -460,7 +460,7 @@ def _heartbeat_manager(self):
         if self._process and not self._process.is_alive():
             self._process.join(timeout=0)
             if not self.done:
-                self.log.warning(
+                self.log.debug(
                     "DagFileProcessorManager (PID=%d) exited with exit code %d - re-launching",
                     self._process.pid, self._process.exitcode
```

Comment:
I think this should be `debug` because as I user I cannot do anything about it

Reply:
It's a sign that things are going wrong though, and it is definitely an exceptional conditions. I would say this should still be a warning

Reply:
Won't there be more meaningful logs before this one?

Reply:
No possibly not,at least not in the Scheduler log files - possibly no where if it dies with OOM or is killed